providers/provider_data.go

```diff
@@ -283,7 +284,12 @@ func (p *ProviderData) buildSessionFromClaims(rawIDToken, accessToken string) (*
 }
 
 func (p *ProviderData) getClaimExtractor(rawIDToken, accessToken string) (util.ClaimExtractor, error) {
-	extractor, err := util.NewClaimExtractor(context.TODO(), rawIDToken, p.ProfileURL, p.getAuthorizationHeader(accessToken))
+	var profileURL *url.URL
+	if !p.SkipClaimsFromProfileURL {
+		profileURL = p.ProfileURL
+	}
```

Comment:
What's your opinion on the following for better readability?

```suggestion
	profileURL := p.ProfileURL
	if p.SkipClaimsFromProfileURL {
		profileURL = &url.URL{}
	}
```

Reply:
Agree. Committing your suggestion.